### PR TITLE
Add serve command

### PR DESCRIPTION
**Pull request by Que3216**
Add command to spin up a localhost server to serve the assets that were snapshotted when running a particular replay.

I added this mainly for my own usage in debugging, but I think it's useful and worth merging. What do you think?

Downside would be more commands get listed when you run `npx meticulous`, which could make it more confusing for users to learn how to use the product. (ideal thing would be to start grouping the commands into core and advanced, not sure that's possible)

### CODE REVIEW

File: packages/cli/src/main.ts [hunk 1]
```diff
@@ -44,6 +45,7 @@ export const main: () => void = () => {
     .command(showProject)
     .command(updateTests)
     .command(uploadBuild)
+    .command(serve)
```
Comment (alexivanov) [inline packages/cli/src/main.ts:48]: Looks like this will hide it from the help menu
```suggestion
    .command("serve", false, serve)
```

**Review by alexivanov — APPROVED** [review]

**Review by leafty — APPROVED** [review]
LGTM

I think maybe this type of utilities should not be added to this repository.

File: packages/cli/src/main.ts (hunk 1 above)
Comment (Que3216) [reply packages/cli/src/main.ts:48]: Oh nice, I'll try that

**Comment by Que3216** [comment]
> I think maybe this type of utilities should not be added to this repository.

You're worried about more bytes for customers to download, or making the CLI help screen more confusing by having lots of junk commands?

We could have a separate debugging cli. Though I think if Alex's hiding suggestion works though then should be OK for commands that are lightweight like this / better than overhead of maintaining two CLIs and sharing the code amongst them. So will test that first.

**Comment by leafty** [comment]
> > I think maybe this type of utilities should not be added to this repository.
> 
> You're worried about more bytes for customers to download, or making the CLI help screen more confusing by having lots of junk commands?
> 
> We could have a separate debugging cli. Though I think if Alex's hiding suggestion works though then should be OK for commands that are lightweight like this / better than overhead of maintaining two CLIs and sharing the code amongst them. So will test that first.

Simply put, this repository is code which we ship to users and is ISC-licensed :) It is easier for us to make internal utilities we move here when needed than try to do the opposite.

File: packages/cli/src/main.ts (hunk 1 above)
Comment (Que3216) [reply packages/cli/src/main.ts]: It works -- thanks!

**Pull request merged**